docs/data-sources/service_object.md

```diff
@@ -7,7 +7,7 @@ data "valtix_service_object" "app1_svc_https" {
   name = "app1-svc-https"
 }
 
-rule_set_id = valtix_policy_rule_set.ingress_policy_rule_set.id
+rule_set_id = valtix_policy_rule_set.ingress_policy_rule_set.id {
```

Comment:
Why is the code like this? Looks its missing a full example

Reply:
Isn't this a datasource that is accessed by a name alone?  All I did was add the opening bracket as that was missing.